rest/route/github.go

```diff
@@ -573,11 +573,25 @@ func (gh *githubHookApi) commitQueueEnqueue(ctx context.Context, event *github.I
 	if projectRef == nil {
 		return errors.Errorf("no project with commit queue enabled for '%s:%s' tracking branch '%s'", userRepo.Owner, userRepo.Repo, baseBranch)
 	}
+
+	patchId, err := gh.sc.AddPatchForPr(ctx, *projectRef, PRNum, cqInfo.Modules, cqInfo.MessageOverride)
```

Comment:
I know you didn't make this variable but, PRNum is kind of upsetting 😅  why is this not prNum? Can we change it?

Reply:
to be fair I think prNum trips one of the linters 😒 changed though

Reply:
whoa i wonder why